test/mocha.opts

```diff
@@ -1,5 +1,5 @@
 --timeout 1200s
 --exit
 --colors
---require ./test/setup.js
+--require test/setup.js
```

Comment:
why did you change here...?

Reply:
To make paths uniform. I didn't use `./` for other paths too.